packages/status-react/src/protocol/use-sorted-chats.tsx

```diff
@@ -0,0 +1,73 @@
+import { useMemo } from 'react'
+
+import { useProtocol } from './provider'
+
+import type { Community } from '@status-im/js'
+
+export type Chat = Community['description']['chats'][0] & {
+  id: string
+}
+
+export type Category = {
+  id: string
+  name: string
+  position: number
+  chats: Chat[]
+}
+
+type Result = {
+  categories: Category[]
+  chats: Chat[]
+}
+
+function sortByPosition<T extends { position: number }>(items: T[]): T[] {
+  items.sort((a, b) => {
+    if (a.position < b.position) {
+      return -1
+    }
+    if (a.position > b.position) {
+      return 1
+    }
+    return 0
+  })
+
+  return items
+}
+
+export const useSortedChats = (): Result => {
```

Comment:
why on the screenshot categories go:

1. test-category-2
2. test-category
3. test-category-3

if 

```
{
  "e09d5568-e158-4c32-9bdf-4d369bea0079": {
    categoryId: "e09d5568-e158-4c32-9bdf-4d369bea0079",
    name: "test-category",
    position: 0,
  },
  "86b29a23-929a-44c2-98f5-e3703a004d92": {
    categoryId: "86b29a23-929a-44c2-98f5-e3703a004d92",
    name: "test-category-3",
    position: 1,
  },
  "8f2cc5d7-ecb6-42cd-af17-1e7ed993bd7a": {
    categoryId: "8f2cc5d7-ecb6-42cd-af17-1e7ed993bd7a",
    name: "test-category-2",
    position: 2,
  },
}
```

?

Reply:
interesting, I've tried all Go nodes and a Nim one, and I'm getting what I've shared above

Reply:
which matches mobile

Reply:
ordering works as expected if both desktop and web have same network (bridged eth and status fleets)

Reply:
mobile matches too, expect it lists categories first